sql/src/main/java/io/crate/operation/collect/MapSideDataCollectOperation.java

```diff
@@ -437,9 +437,7 @@ private int numShards(CollectNode collectNode, String localNodeId) {
                             );
                             shardCollectors.add(collector);
                         } catch (IndexShardMissingException e) {
```

Comment:
can be put into the same catch block... ` catch (IndexShardMissingException | CancellationException e)`